doc/examples/pymvpa2.cfg

```diff
@@ -136,3 +136,6 @@ pimp docstrings = yes
 # Option is in effect at import time, i.e. change of it wouldn't effect after dataset
 # has already being loaded
 repr = full
+
```

Comment:
Hi, 
Totally agree about the py3 versions. I should've gitignore-d them. About the other changes:
** Added the [location] section and 'tutorial data' variable to doc/examples/pymvpa2.cfg since
cfg.get('location', 'tutorial data') asks for them in rsa_fmri.cfg
 ** Added "flavor='25mm' in In rsa_fmri.py, line datapath = load_tutorial_data(...)  since '25mm' was the only mask that came with PyMVPA. Without that option the rsa_fmri.py script fails.
** The line anat = nb.load(pjoin(datapath, 'sub001', 'anatomy', 'highres001.nii.gz')) in rsa_fmri.py asks for the highres image. PyMVPA came with lowres ones only, therefore the file addition.
Cheers!